specification/compute/resource-manager/Microsoft.Compute/stable/2022-04-04/CloudServiceRP/cloudService.json

```diff
@@ -2141,11 +2175,11 @@
         },
         "settings": {
           "description": "Public settings for the extension. For JSON extensions, this is the JSON settings for the extension. For XML Extension (like RDP), this is the XML setting for the extension.",
-          "type": "string"
+          "type": "object"
```

Comment:
@arpitkh12  could you provide justification/reason for breaking change here for the breaking change review board?

Reply:
We have the breaking change because the field return from backend was always a JToken. Historically, this was not being used much or was being set to a string. But now with more extensions being added to CSES and Automatic injection of extension this is impacting customers and has led us to pause rollout of AzSecPack extension. Hence this breaking change

Reply:
@JeffreyRichter  could you please review? even though this PR is merged, it's easier to breaking change here than in our PR to main https://github.com/Azure/azure-rest-api-specs/pull/19354